sync_leader.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	cryptoRand "crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"golang.org/x/crypto/nacl/box"
+)
+
+// leaderSync holds the state and code that we need for a one-off sync with a
+// worker enclave.
+type leaderSync struct {
+	attester
+	keys *enclaveKeys
+}
+
+// asLeader returns a new leaderSync struct.
+func asLeader(keys *enclaveKeys) *leaderSync {
+	return &leaderSync{
+		attester: &dummyAttester{},
```

Comment:
Is this supposed to default to a dummyAttester here?

Reply:
Right, that's a debug artifact. Removed in 35d6911.